libconcorde_tsp_solver/package.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<package format="2">
+	<name>libconcorde_tsp_solver</name>
+	<version>0.6.10</version>
+	<description>Wrapper for the concorde traveling salesman problem solver. The code was obtained from http://www.math.uwaterloo.ca/tsp/concorde/downloads/downloads.htm all rights of it go to the corresponding authors David Applegate, Robert Bixby, Vasek Chvatal and William Cook. 
+The library doesn't give a specific license, but is provided free for academic research use, for further licensing options contact William Cook.
```

Comment:
alright, fine for me.

Maybe you can just fix the indentation and line lengths (below as well) but this is very minor and I'll let someone else decide wether to merge this as is or not :-)